linfa-supervised/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "linfa-supervised"
+version = "0.1.0"
+authors = ["Khalil HADJI <khalil.hadji@outlook.fr>"]
+edition = "2018"
+description = "A collection of supervised learning algorithms"
+license = "MIT/Apache-2.0"
+
+repository = "https://github.com/LukeMathWalker/linfa"
+
+keywords = ["supervised", "machine-learning", "linfa", "regression", "linear", "ridge", "lasso"]
+categories = ["algorithms", "mathematics", "science"]
+
+[dependencies]
+ndarray = { version = "0.13" , features = ["rayon"] }
+ndarray-linalg = { version = "0.12", features = ["openblas"] }
```

Comment:
We don't want to force the usage of a certain BLAS implementation over the other ones (e.g. `intel-mkl`) - I would suggest to use a features-strategy similar to what I put down here: https://github.com/rust-ndarray/ndarray-examples/blob/d7d43e018cade1f8bf0d92220081ba66963bab07/linear_regression/Cargo.toml#L8